### PR TITLE
Semantic cleanup

### PR DESCRIPTION
**Pull request by TheBB**

### CODE REVIEW

File: layers/+lang/c-c++/packages.el [hunk 1]
```diff
@@ -108,8 +108,8 @@
     (spacemacs/helm-gtags-define-keys-for-mode 'c++-mode)))
 
 (defun c-c++/post-init-semantic ()
-  (semantic/enable-semantic-mode 'c-mode)
-  (semantic/enable-semantic-mode 'c++-mode))
+  (add-hook 'emacs-lisp-mode-hook 'c-mode)
```
Comment (fbergroth) [inline layers/+lang/c-c++/packages.el]: Some delicious copy pasta here :smile:

**Comment by syl20bnr** [comment]
Great, thank you! :+1: 
I fixed the typo mentioned by @tuhdo 
Cherry-picked.

**Comment by TheBB** [comment]
Ah, because I'm too quick for my own good.

**Comment by fbergroth** [comment]
I'm amazed. I thought I pointed with pretty big fingers.

**Pull request closed without merging**